lib/SILOptimizer/Utils/Differentiation/JVPEmitter.cpp

```diff
@@ -1,4 +1,4 @@
-//===--- JVPEmitter.h - JVP Generation in Differentiation -----*- C++ -*---===//
+//===--- JVPEmitter.cpp - JVP Generation in Differentiation ---*- C++ -*---===//
```

Comment:
```suggestion
//===--- JVPEmitter.cpp - JVP generation in differentiation ---*- C++ -*---===//
```

Please use consistent sentence casing in all headings.

Reply:
Done.